dll/win32/shell32/dialogs/filetypes.cpp

```diff
@@ -1735,24 +1957,38 @@ FolderOptionsFileTypesDlg(
             lppl = (LPNMLISTVIEW) lParam;
             switch (lppl->hdr.code)
             {
+                case LVN_GETDISPINFO:
+                {
+                    NMLVDISPINFO *pLVDI = (NMLVDISPINFO*)lParam;
+                    PFILE_TYPE_ENTRY p = (PFILE_TYPE_ENTRY) pLVDI->item.lParam;
```

Comment:
```suggestion
                    PFILE_TYPE_ENTRY p = (PFILE_TYPE_ENTRY)pLVDI->item.lParam;
```
and "p" --> "pEntry"